spec/rspec/core/configuration_spec.rb

```diff
@@ -1229,7 +1229,171 @@ def metadata_hash(*args)
       end
     end
 
+    describe "color options" do
+      before { config.no_color = false }
+
+      context "#force_color=false, #no_color=false" do
+        context "with output.tty?" do
+          it "sets color_enabled?" do
+            output = StringIO.new
+            config.output_stream = output
+
+            allow(config.output_stream).to receive_messages :tty? => true
+
+            expect(config.color_enabled?).to be_truthy
```

Comment:
I'd prefer `be true` over `be_truthy` and `be false` over `be_falsey` -- mind updating the expectations?